MovieGPT-api/src/main/java/lt/techin/group/project/security/TestController.java

```diff
@@ -0,0 +1,25 @@
+package lt.techin.group.project.security;
+
```

Comment:
Will we keep test controller here only till development finishes? Because no tests should be in main package.

Reply:
My bad, I left some junk. I was testing if the token was accepted correctly. Removing on next pull request.